package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@interlay/interbtc-api",
-  "version": "1.9.0",
+  "version": "1.9.1",
```

Comment:
Might need to refresh from master. Currently, the version on master is already 1.9.1